### ISSUE TITLE
i have already installed jupyterlab-vimrc, but i can't find vimrc in my jupyterlab

### ISSUE THREAD

**Issue by ZihaoXingUP** [ianhi/jupyterlab-vimrc#14]
![image](https://user-images.githubusercontent.com/68001792/98501598-a8566e80-228a-11eb-826c-9a4c0823bdaf.png)
![image](https://user-images.githubusercontent.com/68001792/98501675-e05db180-228a-11eb-8e1b-9e2c74541fe8.png)

**Comment by ianhi** [comment]
Hi @ZihaoXingUP it looks as though you installed using `pip` which is what is necessary to install for jlab3. For jupyterlab 2 you can still install it using:
```
jupyter labextension install jupyterlab-vimrc
```

**Comment by ZihaoXingUP** [comment]
yes ,i use   'jupyter labextension install jupyterlab-vimrc' to install it ,but there are some errors in my computer.
| |
邢自豪
|
|
zihaoxingstudy@163.com
|
签名由网易邮箱大师定制
On 11/9/2020 13:24，Ian Hunt-Isaak<notifications@github.com> wrote：

Hi @ZihaoXingUP it looks as though you installed using pip which is what is necessary to install for jlab3. For jupyterlab 2 you can still install it using:

jupyter labextension install jupyterlab-vimrc

**Comment by ZihaoXingUP** [comment]
> Hi @ZihaoXingUP it looks as though you installed using `pip` which is what is necessary to install for jlab3. For jupyterlab 2 you can still install it using:
> 
> ```
> jupyter labextension install jupyterlab-vimrc
> ```

yes ,i use jupyter labextension install jupyterlab-vimrc to install it ,but meet some errors
![image](https://user-images.githubusercontent.com/68001792/98510736-94b50300-229e-11eb-9d02-510e8b9757f2.png)

**Comment by ianhi** [comment]
Oh weird. I think that's an issue with how jupyterlab handles non-utf encodings. It looks the same as this issue: https://github.com/jupyterlab/jupyterlab/issues/8600

Can you try to use the workaround there and see if that fixes it?
